### PR TITLE
add Recurrent Models and Examples with MXNetR post

### PR DESCRIPTION
**Pull request by ziyeqinghan**
Add a post for RNN models with MXNet in R.

### CODE REVIEW

**Comment by tqchen** [comment]
please add tag of rstats as in other blogs so that it is directly pushed to r-blogger. Thanks!

File: _posts/2016-08-16-recurrent-models-and-examples-with-mxnet-r.markdown [hunk 1]
```diff
@@ -0,0 +1,512 @@
+---
+layout: post
+title:  "Recurrent Models and Examples with MXNetR"
+date:   2015-08-16 14:41:00 -0800
+author: Yuqi Li
+categories: mxnet
+comments: true
+---
+
```
Comment (tqchen) [inline _posts/2016-08-16-recurrent-models-and-examples-with-mxnet-r.markdown:9]: Give an brief introduction to MXNetR package

**Comment by ziyeqinghan** [comment]
tag of rstats is added and a brief introduction to MXNetR package is given.

**Comment by tqchen** [comment]
Thanks, one final comment, please add an running example on the end, for example training char-rnn and generate some results.

**Comment by ziyeqinghan** [comment]
In the chapter "RNN Example", the link to the documentation of `char-rnn` example is showed. Should I rewrite it in this post?

**Comment by tqchen** [comment]
yes, paste a snippet here will be good

**Comment by ziyeqinghan** [comment]
The documentation of `char-lstm` example is added.

**Comment by tqchen** [comment]
Thanks for the great job! this is merged

**Pull request merged**